build_defs/cpp_opts.bzl

```diff
@@ -39,6 +39,9 @@ LINK_OPTS = select({
         "-lm",
         "-framework CoreFoundation",
     ],
+    "@platforms//os:windows": [
+        "-ldbghelp",
```

Comment:
We should duplicate the default behavior here.  I suspect this will break something do to missing pthread/m

Reply:
Good point. Done.